packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/WebhookForm/tests/index.test.js

```diff
@@ -70,11 +69,11 @@ describe('Create Webhook', () => {
 
     render(App);
 
-    userEvent.type(screen.getByLabelText(/name/i), 'My webhook');
-    userEvent.type(screen.getByLabelText(/url/i), 'https://google.fr');
+    fireEvent.change(screen.getByLabelText(/name/i), { target: { value: 'My webhook' } });
+    fireEvent.change(screen.getByLabelText(/url/i), { target: { value: 'https://google.fr' } });
     fireEvent.click(screen.getByRole('checkbox', { name: /entry.create/i }));
 
-    userEvent.click(screen.getByRole('button', { name: /Save/i }));
+    fireEvent.click(screen.getByRole('button', { name: /Save/i }));
```

Comment:
sorry same comment 🤦‍♂️ ? can you not use userEvent? I know that kent c dodds recommends it in his article https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

Reply:
Let's keep one discussion about that open.